app/lib/shared/markdown.dart

```diff
@@ -194,7 +201,10 @@ class _UnsafeUrlFilter implements m.NodeVisitor {
       element.attributes.remove(attr);
       return true;
     }
-    if (uri.shouldIndicateUgc) {
+
+    /// <img src=""> elements are not covered by package:sanitize_html.
+    /// TODO: decide to remove from here or add it to sanitize_html.
+    if (tag == 'img' && uri.shouldIndicateUgc) {
       element.attributes['rel'] = 'ugc';
     }
     return false;
```

Comment:
Do we need `rel="ugc"` on `<img` tags?

Reply:
Hm, actually, we probably shouldn't have `rel` attribute on `img` tags, as it is not a valid attribute on them. I'll update the PR.